angular-frontend/src/assets/auth.conf.json

```diff
@@ -0,0 +1,5 @@
+{
+  "authority": "<your-authserver-uri>",
+  "clientId": "<your-client-id>",
+  "scope": [ "openid", "profile", "email", "message.read", "message.write" ]
```

Comment:
**[suggestion]**: this most likely only needs to be just `openid` and `email`, for the purposes of the accelerator.

Reply:
I'm making some changes to have the java-rest-sevice be a resource server. For the integration with that we need `message.read` and `message.write`. My understanding is that these 2 accelerators are already closely related.

Reply:
Gotcha.